wagtail/utils/deprecation.py

```diff
@@ -2,5 +2,8 @@ class RemovedInWagtail12Warning(DeprecationWarning):
     pass
 
 
+removed_in_next_version_warning = RemovedInWagtail12Warning
```

Comment:
``DeprecationWarning`` could be used by other third party apps/modules and we're only concerned about Wagtails deprecation warnings.

Django have taken a similar approach, possibly to reduce noise I guess.